Changed login url

JensdeVlaming opened the pull request:
Error resolved which was introduced because of an URL change by KPN.

OlafHaalstra commented:
Hi Jens, 

Thanks for your PR. I just updated the URL myself and after checking if everything was committed correctly I saw that you also fixed it! Thanks a lot, sorry for not merging earlier.

OlafHaalstra commented:
Closes #5

JensdeVlaming commented:
> Hi Jens, 
> 
> 
> 
> Thanks for your PR. I just updated the URL myself and after checking if everything was committed correctly I saw that you also fixed it! Thanks a lot, sorry for not merging earlier. 

No problem! Glad you got it fixed.
